src/decisionengine/framework/modules/de_logger.py

```diff
@@ -51,11 +51,12 @@ def set_logging(
         logger.debug('Reusing existing logging handlers')
         return None
 
-    logconf = copy.deepcopy(configDict.pylogconfig)
+    #logconf = copy.deepcopy(configDict.pylogconfig)
+    logconf = configDict.pylogconfig
```

Comment:
Similar comments on copy

Reply:
Will fix.

Reply:
Pat and I reached an agreement on the logging config dict. I defined the dict to be global in logging_configDict.py, which is how it has been defined in my head. WIth this, none of the copying, etc is needed.